app/models/activity.rb

```diff
@@ -21,7 +21,7 @@ class Activity < ApplicationRecord
     fund: "fund",
     programme: "programme",
     project: "project",
-  }, _prefix: :is, _suffix: :level
+  }
```

Comment:
Aww no @mec will be sad to see this go 😆

Reply:
I noticed the differences of approaches during a conflict nightmare last night! Given I had used the `fund?`, `programme?` in favour of using the enums which I thought was the plan, then I rebased ontop of `project?` and it all went wrong and I had to pick. I chose the one that following the enum conventions and removed the extra tests. 

I can change it back as a follow up refactor commit if this way provides more clarity?

Reply:
I think using `activity.project?` etc is clearer myself (and less code 😆 )

Reply:
😄 I like the wordy version but happy to go without! @lozette you should definately call me out on things you have a preference for!

Reply:
I thought the prefix + suffix thing was pretty cool tho!